### PR TITLE
Added research papers related to CNN used in machine translation

### PR DESCRIPTION
**Pull request by the-ethan-hunt**
I have added three research papers for issue #92 - 
- Two Facebook AI research papers for CNN in machine translation. The code is also available
- A research paper from University of Amsterdam that compares CNN with RNN encoders.(code unavailable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -249,6 +249,9 @@ timesteps, thereby achieving strong performance in many text classification task
...
 * [arXiv: Neural Machine Translation by jointly learning to align and translate](http://arxiv.org/pdf/1409.0473v6.pdf)
 Bahdanau, Cho 2014 introduced the **attention mechanism** in NLP
+* [arXiv: A Convolutional encoder model for neural machine translation](https://arxiv.org/pdf/1611.02344.pdf) by Gehring,Auli,Grangier and Dauphin 2017. The paper is from Facebook AI research and its code is available [here](https://github.com/facebookresearch/fairseq).
+* [Convolutional over Recurrent Encoder for neural machine translation](https://ufal.mff.cuni.cz/pbml/108/art-dakwale-monz.pdf) by Dakwale and Monz from University of Amsterdam compare the CNNs with a recurrent neural network with additional convolutonal layers.
+* [Convolutional Sequence to Sequence learning](https://arxiv.org/pdf/1705.03122.pdf) by Gehring,Auli,Grangier and Dauphin 2017. The paper is from Facebook AI research and its code is available [here](https://github.com/facebookresearch/fairseq).
```
Comment (NirantK) [inline README.md]: Few quick suggestions to make your PR go from good to `awesome`: 

1. Spaces between comma separated words: ```Gehring,Auli,Grangier and Dauphin``` is better as ```Gehring, Auli, Grangier and Dauphin``` or simply ```Gehring et al, 2017```

2. Is the Dakwale et al work a comparison paper or a different architecture? From what I read, it seems like they've proposed a new augmented RNNs + CNN architecture. Please check and update according to your judgement?

Comment (the-ethan-hunt) [reply README.md]: @NirantK , sure, that would seem more 'ordered'. I have made a commit according to your suggestions.
Talking of the Dakwale et al paper, they have also compared it to a deep RNN encoder too.(See Section 6). They have proposed a different architecture and compared it. Any suggestions?

**Comment by NirantK** [comment]
Good stuff @the-ethan-hunt ! Thanks for the contribution! 

Are there any other sections you'd like to help revamp or create? We do not cover text summarization (refer #85) at all. Would you like to create an entire section? 

Good chance to set the foundation!

**Comment by the-ethan-hunt** [comment]
@NirantK , sure! I am kind of enjoying myself my contributing to this awesome list!
I would cover text summarization in another PR.

**Pull request merged**